Content.Shared/GameObjects/Components/Movement/SharedClimbingComponent.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+using System;
+
+namespace Content.Shared.GameObjects.Components.Movement
+{
+    public class SharedClimbingComponent : Component
+    {
+        public sealed override string Name => "Climbing";
+        public sealed override uint? NetID => ContentNetIDs.CLIMB_MODE;
```

Comment:
This name of this net id should be changed to `CLIMBING` as the name of the component was changed in https://github.com/space-wizards/space-station-14/pull/1750/commits/ae4fc15bcc39df8e3eff8e4e42cdda5131551133